Tests/SentryTests/Integrations/SessionReplay/SentrySessionReplayIntegrationTests.swift

```diff
@@ -32,28 +31,38 @@ class SentrySessionReplayIntegrationTests: XCTestCase {
         clearTestState()
     }
     
-    func startSDK(sessionSampleRate: Float, errorSampleRate: Float, enableSwizzling: Bool = true) {
+    private func getSut() -> SentrySessionReplayIntegration? {
+        guard let sut = SentrySDK.currentHub().installedIntegrations().first as? SentrySessionReplayIntegration else {
+            XCTFail("Did not installed replay integration")
+            return nil
+        }
+        return sut
```

Comment:
`l`: You could also use `XCTUnwrap` to avoid returning `nil`, but it will change this method to a `throws`.

Reply:
true. I mostly forget this. I think its a good idea because it will increase coverage.